packages/integration/src/awsS3/config.ts

```diff
@@ -46,6 +48,16 @@ export type AwsS3IntegrationConfig = {
    * roleArn
    */
   roleArn?: string;
+
+  /**
+   * validateHost
+   */
+  validateHost: boolean;
+
+  /**
+   * ssl
+   */
+  ssl: boolean;
```

Comment:
```suggestion
  ssl?: boolean;
```

Reply:
Default values (`true`) for these flags are implemented similarly to how it works for `host`, which is not a nullable string, but defaults to `AMAZON_AWS_HOST` in `readAwsS3IntegrationConfigs()` if not supplied.

Reply:
I think the terms TLS or HTTPS are preferred over SSL. What do you think about flipping this around as well to make it more natural with the default to true, that is have the config be an optional `noHttps` or something of that sort?

Reply:
Per @freben's suggestion, changed `validateHost`/`ssl` to `endpoint`/`s3ForcePathStyle` config.